### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
- #5080
- #5065
- #5064
- #5055
- #5049
- #5045
- #5036
- #5043
- #5042

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5082) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5082) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: LucasCalazans<br/>:white_check_mark: gerrits-ecommerce<br/>:white_check_mark: dobooth<br/>:white_check_mark: serhiyzhovnir<br/>:white_check_mark: diazwatson<br/>:white_check_mark: gulshankumar<br/>:x: kandy

File: guides/v2.2/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -204,7 +207,7 @@ The `{{site.data.var.ct}}` version 2002.0.18 and later supports developer mode.
 
 1.  _Optional_: Configure the Docker global variables in the `docker-compose.yml` file. For example, you can [enable and configure Xdebug]({{ page.baseurl }}/cloud/docker/docker-development-debug.html).
 
-1.  Start the file synchronization.
+1.  Start the file synchronization (Use one of the following).
```
Comment (erikmarr) [inline guides/v2.2/cloud/docker/docker-config.md]:
```suggestion
1.  Start the file synchronization (use one of the following).
```

File: guides/v2.2/ui_comp_guide/components/ui-draganddrop.md [hunk 2]
```diff
@@ -7,41 +7,15 @@ The DragAndDrop component is an [extension](https://glossary.magento.com/extensi
...
-    <td><code>true</code></td>
-  </tr>
-  <tr>
-    <td><code>noSelectClass</code></td>
-    <td>CSS class applied to the table when one of its columns is being dragged.</td>
-    <td>String</td>
-    <td><code>_no-select</code></td>
-  </tr>
-</table>
+|Option|Description|Type|Default Value|
```
Comment (erikmarr) [inline guides/v2.2/ui_comp_guide/components/ui-draganddrop.md]: any reason why this table is formatted differently than the ui-bookmarks.md above?

Comment (dobooth) [reply guides/v2.2/ui_comp_guide/components/ui-draganddrop.md]: I think the closing | is optional.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**